static/app/views/replays/detail/console/index.tsx

```diff
@@ -30,10 +41,8 @@ function Console({breadcrumbs, startTimestamp = 0}: Props) {
     () =>
       !searchTerm && logLevel.length === 0
         ? breadcrumbs
-        : breadcrumbs.filter(
-            breadcrumb =>
-              breadcrumb.message?.toLowerCase().includes(searchTerm) &&
-              logLevel.includes(breadcrumb.level)
+        : breadcrumbs.filter(breadcrumb =>
+            filterBreadcrumbs(breadcrumb, searchTerm, logLevel)
```

Comment:
I would probably change the function signature to be `filterBreadcrumbs(breadcrumbs: Crumb[], ...)` and include the logic from the ternary. That way we can add test cases for those conditions as well.

Reply:
Good call, adjusted.